concordia/static/js/action-app/components.js

```diff
@@ -784,45 +784,26 @@ class ImageViewer {
     }
 
     update(asset) {
-        let tileSources = [
+        let tileSource = [
```

Comment:
I think this could technically not be an array when there's only one source (e.g. `tileSource = {"type": "image", …};`) but I'm not sure it's worth touching.